labeller/frontend/index.js

```diff
@@ -117,6 +112,36 @@ const handleSaveButtonClicked = async () => {
   }
 };
 
+const handleDeleteButtonClicked = async () => {
+  const videoPath = $("#videoList").val();
+  if(!videoPath) {
+    return alert("No video has been selected");
+  }
+
+  const selectedVideoIndex = $("#videoList").prop("selectedIndex");
+  const {videoUrl, annotationInfo} = vidUrls_annotInfos[selectedVideoIndex];
+  if (!annotationInfo) {
+    return alert("No saved markers to delete")
+  }
+
+  if (!prompt("Are you sure? (Type anything and press enter for yes)")) {
```

Comment:
```suggestion
  if(!confirm("Are you sure?")) {
```

Reply:
disagree. what if they type 'no' or 'NO'. maybe I should just make it `type 'yes' to confirm`